strings/aho_corasick.py

```diff
@@ -1,10 +1,17 @@
 from collections import deque
-from typing import Dict, List, Union
+from typing import Deque, TypedDict, Union
+
+
+class type_of_dict(TypedDict):
+    value: str
+    next_states: list[int]
+    fail_state: int
+    output: list[str]
 
 
 class Automaton:
-    def __init__(self, keywords: List[str]):
```

Comment:
Use these (type check time constructs) instead of runtime constructs.

Reply:
I'm not sure I understand what you're trying to say. Could you please explain it a bit?

Reply:
use `List` instead `list`. `List` is used during check time. Both syntax achieve the same thing. It's neat to use `List` besides.

Reply:
https://stackoverflow.com/a/62033243

Also in the docs, the generic types from typing have been deprecated for Python3.9